src/main/webapp/Index.html

```diff
@@ -10,37 +10,46 @@
     <script src="https://kit.fontawesome.com/ce4cad5060.js" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="Style.css">
+    <link rel="stylesheet" href="CSS/style1.css">
     <title>Welcome Admin</title>
 </head>
 
-<body>
+<body class="bg-light">
    
     <!-- This code is for side-navigation bar -->
-    <nav class="navbar navbar-expand d-flex flex-column align-item-start bg-dark" id="sidebar">
-        
-            <h3 class="container-fluid font-weight-bold text-light justify-content-center position-relative text-center">AdminLTE</h3>
+    <nav class="navbar navbar-expand d-flex flex-column align-item-start bg-dark zi1" id="sidebar">
+        <div class="container-fluid  justify-content-center adminst headcolor">
+            <h3 class="font-weight-bold text-light justify-content-center position-relative text-center ">AdminLTE</h3>
+        </div>
   
         <ul class="navbar-nav d-flex flex-column mt-3 w-100">
             <li class="nav-item w-100 "> 
-            	<img src="image/vk.jpg" alt="Avatar" class="avatar bg-light margin-left" > 
-            		<a href="#profile" style="color: #ffffff; text-decoration: none; margin-left: 0.625rem;">Alexander Pierce</a>
+            	<img src="image/vk.jpg" alt="Avatar" class="bg-light margin-left rounded-circle direct-chat-img mt-3" > 
+            		<a href="#profile" style="color: #ffffff; text-decoration: none; margin-left: 0.625rem;" class="fs-5">Alexander Pierce</a>
```

Comment:
Don't add style directly. Instead of that create a custom class and use it here.